src/modules/esl-utils/dom/rect.ts

```diff
@@ -112,4 +127,11 @@ export class Rect {
   public shrink(decrementX: number, decrementY: number = decrementX): Rect {
     return this.grow(-decrementX, -decrementY);
   }
+
+  /**
+   * Method to check if element's area is bigger than zero
+   */
+  public isEmpty(): boolean {
+    return this.height === 0 || this.width === 0;
+  }
```

Comment:
Recommended proposal:
```suggestion
  /** @returns the rectangle area square */
  public get square(): boolean {
    return this.height * this.width;
  }
```
With the consumption:
```
 rect.square === 0 // instead rect.isEmpty()
```

As an alternative:
```suggestion
  /** @returns if the element's area is bigger than zero */
  public get empty(): boolean {
    return this.height === 0 || this.width === 0;
  }
```